src/BizHawk.Emulation.Cores/Consoles/Nintendo/N64/NativeApi/mupen64plusCoreApi.cs

```diff
@@ -904,13 +904,13 @@ public void LoadSaveram(byte[] src)
 			m64pload_saveram(src);
 		}
 
-		/* TODO: Support address masks and null address */
+		/* TODO: Support address masks */
 		public void SetBreakpoint(BreakType type, uint? address)
 		{
 			m64p_breakpoint breakpoint = new m64p_breakpoint
 			{
-				address = address.Value,
-				endaddr = address.Value + 0x03,
+				address = address ?? 0x0, // If address is null, max address range to match any address
+				endaddr = address != null ? address.Value + 0x03 : uint.MaxValue,
 				flags = (uint)m64p_dbg_bkp_flags.M64P_BPT_FLAG_ENABLED
 			};

```

Comment:
I think this would be clearer with the null check as the outermost expression i.e.
```csharp
m64p_breakpoint breakpoint = address is null
	? new() { ... }
	: new() { ... };
```

Reply:
I agree. Hopefully I've captured the style you were going for.